### PR TITLE
[full-ci] Using auth token by default

### PR DESCRIPTION
**Pull request by ScharfViktor**
We've switched to using an api token, and now I make using it by default

### CODE REVIEW

**Review by saw-jan — COMMENTED** [review]
So, we don't support basic auth in e2e now?

**Comment by saw-jan** [comment]
> Yes, I was doubting whether to keep it and decided to delete. Do you suggest to leave?

we are token or basic auths to setup the server (users/groups/resource) so how we do it should not matter. Token auth is needed for some servers where we cannot enable basic auth.

Using basic auth is straightforward and should be less overhead to the execution time than token auth. It's because, to access the token we do admin login via webUI which we don't have to do if we use basic auth.
So for this, even though how we do it doesn't matter, I would prefer token auth to be an option not mandatory.

I see that we are running tests with token auth in CI as well. But any reasons why we switched to token auth?

File: tests/e2e/support/api/http.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const request = async ({
     Authorization: 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
   }
 
-  if (config.apiToken) {
+  if (!config.basicAuth) {
     authHeader.Authorization = 'Bearer ' + tokenEnvironment.getToken({ user }).accessToken
   }
```
Comment (saw-jan) [inline tests/e2e/support/api/http.ts:30]: swap it like:
```js
const authHeader = {
    Authorization: 'Bearer ' + tokenEnvironment.getToken({ user }).accessToken
  }

  if (config.basicAuth) {
    authHeader.Authorization = 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
  }
```

Comment (ScharfViktor) [reply tests/e2e/support/api/http.ts:30]: it doesn't work. if `BASIC_AUTH=true` token is undefined

Comment (saw-jan) [reply tests/e2e/support/api/http.ts:30]: Ah, when `BASIC_AUTH=true`, we don't set `accessToken`

**Review by saw-jan — APPROVED** [review]
LGTM :+1:

**Pull request merged**